brave/brave-core
# feat: show token prices in fund wallet screen

Resolves https://github.com/brave/brave-browser/issues/25861

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Create a wallet
2. Navigate to `brave://wallet/crypto/fund-wallet`
3. Observe that prices are shown for supported assets

https://user-images.githubusercontent.com/30185185/194639509-c8959db9-c7a7-4eab-9076-45fbaeab9cb8.mov
